quixstreams/sources/__init__.py

```diff
@@ -2,3 +2,4 @@
 from .manager import SourceException
 from .multiprocessing import multiprocessing
 from .csv import CSVSource
+from .kafka import KafkaSource, QuixEnvironmentSource
```

Comment:
`QuixEnvironmentSource` should be imported from the `quix` package

Reply:
Both the KafkaSource and QuixEnvSource are coming from the `kafka` package. I don't think we should split it since QuixEnvSource inherit from KafkaSource